src/plugins/gui/scenes/FiatPluginWebView.tsx

```diff
@@ -14,9 +16,13 @@ export interface FiatPluginOpenWebViewParams {
 interface Props extends EdgeSceneProps<'guiPluginWebView'> {}
 
 export function FiatPluginWebViewComponent(props: Props): JSX.Element {
-  const { route } = props
+  const { route, navigation } = props
```

Comment:
changing this file is unnecessary. This is only launched after we've launched a fiatPlugin which is already blocked